lib/reporting/reports/list.rb

```diff
@@ -2,7 +2,7 @@
 
 module Reporting
   module Reports
-    class List
+    class List # rubocop:disable Metrics/ClassLength # Because it's a simple class
```

Comment:
There should be  `# rubocop:enable Metrics/ClassLength` at the end to renable the cop

Reply:
That's only needed when the comment is on its own line. Inline comments don't need deactivating.

Reply:
TIL :+1: